azuracast/liquidsoap_custom.liq

```diff
@@ -370,14 +349,6 @@ sources = [
     ({5w and 22h-22h58m}, once(playlist_ganzfeld)),
     ({5w and 20h30m-22h30m}, playlist_off_air_ambient),                 #ambient: moneyka - Ganzfeld
     ({6w and 10h-11h58m}, once(playlist_superstar)),                       #Saturday
-    ({6w and 12h-13h58m}, once(playlist_random_times)),
-    ({6w and 14h-14h58m}, once(playlist_esforditva)),
-    ({6w and 15h-17h58m}, once(playlist_torso)),
-    ({6w and 18h-19h58m}, once(playlist_rnr666)),
-    ({6w and 20h-21h58m}, once(playlist_mood_sequence)),
-    ({6w and 22h-22h58m}, once(playlist_blr)),
-    ({6w23h-7w00h28m}, once(playlist_d23)),
-    ({6w10h30m-7w}, playlist_off_air_ambient),                    #ambient: superstar - D23
```

Comment:
@tuz666 Next time we can try keeping the ambient playlist as a fallback in case of network interruption to the live stream